utils/exporters/blender/addons/io_three/constants.py

```diff
@@ -329,3 +329,4 @@
 
 DOUBLE_SIDED = 'doubleSided'
 
+EXPORT_SETTINGS_KEY = 'three_export_settings'
```

Comment:
Not too much of an issue but an FYI that even tho the constants are snake_case I try to keep the actual strings camelCase for consistency.